merkledag/utils/utils.go

```diff
@@ -153,3 +148,32 @@ func rmLink(ctx context.Context, ds dag.DAGService, root *dag.Node, path []strin
 
 	return root, nil
 }
+
+func (e *Editor) WriteOutputTo(ds dag.DAGService) error {
+	return copyDag(e.GetNode(), e.ds, ds)
```

Comment:
i think this needs what i wrote in https://github.com/ipfs/go-ipfs/commit/978c9fa16fe47ee3c5d524241202e74edec82147#diff-bd0ca9f1e2d5c7ba128af01ec4c1131cR314